.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,35 @@
+// For format details, see https://aka.ms/vscode-remote/devcontainer.json or the definition README at
```

Comment:
What is this used for?

Reply:
It's a VS code dependency and should not have made it in to the commit.  It should be removed.